src/sentry/migrations/0230_auto__del_field_eventmapping_group__del_field_eventmapping_project__ad.py

```diff
@@ -0,0 +1,472 @@
+# -*- coding: utf-8 -*-
```

Comment:
Bleh, nevermind. I guess you'll need it since it'll just continue to be try and add it every time in the future.